contribute/developer-guide.md

```diff
@@ -2,8 +2,6 @@
 
 This guide helps you get started developing Grafana.
 
-Before you begin, you might want to read [How to contribute to Grafana as a junior dev](https://medium.com/@ivanahuckova/how-to-contribute-to-grafana-as-junior-dev-c01fe3064502) by [Ivana Huckova](https://medium.com/@ivanahuckova).
-
```

Comment:
Ivana's article is from several years ago, and isn't helpful anymore -- her advice was to find another new Grafana dev who's interested in writing an updated version we can link here.